packages/modules/src/utils/Utils.ts

```diff
@@ -0,0 +1,76 @@
+import { defaultAbiCoder, keccak256 } from 'ethers/lib/utils'
+import { UserOperation } from '@biconomy/core-types'
+
+export function getUserOpHash(
+  op: UserOperation,
+  entryPointAddress: string,
+  chainId: number
+): string {
+  const userOpHash = keccak256(packUserOp(op, true))
+  const enc = defaultAbiCoder.encode(
+    ['bytes32', 'address', 'uint256'],
+    [userOpHash, entryPointAddress, chainId]
+  )
+  return keccak256(enc)
+}
+
+export function packUserOp(op: UserOperation, forSignature = true): string {
```

Comment:
why getUserOpHash and packUserOp methods are in modules utils?? They should be in SA utils.

Reply:
we use it here only. SA package makes read call in the SDK. I can move these to common package maybe